sympy/utilities/lambdify.py

```diff
@@ -450,6 +452,15 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def print_dummify(expr):
```

Comment:
`print` is a bad name here, as it isn't printing anything. `should_dummify` is better.